Library/Homebrew/test/utils/inreplace_spec.rb

```diff
@@ -52,4 +52,18 @@
       end.to raise_error(Utils::Inreplace::Error)
     end
   end
+
+  describe "#gsub!" do
+    it "substitutes pathname within file" do
+      # For a specific instance of this, see https://github.com/Homebrew/homebrew-core/blob/a8b0b10/Formula/loki.rb#L48
+      described_class.inreplace(file.path) do |s|
+        s.gsub!(Pathname("b"), "f")
+      end
```

Comment:
the "after" seems to also be a Pathname:

```rb
s.gsub! var, testpath
```

Reply:
TY! PTAL.